en/deploy-on-aws-eks.md

```diff
@@ -242,6 +242,10 @@ Allow the bastion host to access the Internet. Select the correct key pair so th
 
 After the bastion host is created, you can connect to the bastion host via SSH and access the TiDB cluster via the MySQL client.
 
+> **Note:**
+>
+> By default, the configuration in `tidb-cluster.yaml` sets up the LoadBalancer for TiDB with the "internal" scheme. This means that the LoadBalancer is only accessible within the VPC, not externally. To access TiDB over the MySQL protocol, you need to use a bastion host or use `kubectl port-forward`. If you want to expose TiDB over the internet and if you are aware of the risks of doing this, you can change the scheme for the LoadBalancer from "internal" to "internet-facing" in the `tidb-cluster.yaml` file, and then re-deploy the loadbalancing service.
```

Comment:
According to @DanielZhangQD, users are not recommended to update "internal" to "internet-facing" directly in the tidb-cluster.yaml file after the deployment. Could we move the note to the Deploy steps, just after the "Refer to configure the TiDB cluster to further customize and configure the CR before applying." sentence? Then users will know this "internal" scheme and can determine whether to change it to "external" during the configuration.